### DISCUSSION TITLE
Issue importing .csv

### DISCUSSION THREAD

**Discussion by IronFixXxeR** [NickvisionApps/Denaro#105] [General]
When importing a 30 day records .csv file from my bank account it says "Imported 0 transactions from CSV". Is this feature not working right now?

**Comment by fsobolev** [comment]
CSV is not a format for storing banking information, it's just a table. CSV Import/Export feature is here mostly for cases when you want to edit Money account in other apps like LibreOffice. A CSV file from your bank will be 100% incompatible with Money, because Money stores its information in CSV file like it needs, not like it's expected by your bank.
There are people in community (myself included) who want to create external tool to convert files from different banks to either `.nmoney` format or CSV that can be imported in Money. But the work on it didn't start yet and for now nobody can say when it will be finished. Feel free to join [our matrix channel](https://matrix.to/#/#nickvision:matrix.org) to participate in development discussion.
